### ISSUE TITLE
No reinicia el minado después de una desconexión a Internet

### ISSUE THREAD

**Issue by Kanopola** [vertcoin-project/one-click-miner-vnext#627]
¡Hola!

Por favor, corrijan este problema de tener que darle a "empezar a minar" o "reintentar" después de una desconexión de Internet. En realidad no sé cómo pueden decir "final" a una App que aún le faltan detalles tan importantes y sencillos como estos.

![Screenshot_8](https://github.com/vertcoin-project/one-click-miner-vnext/assets/56226178/2516543f-0ffa-47d4-949c-811d51ff00e8)

¡Saludos!

**Comment by KforG** [comment]
Effectively a duplicate of #408

Please keep issues in English.

**Comment by KforG** [comment]
Did OCM work for you before the the internet connection was cut? This error suggest that the miner binaries are removed (by antivirus most likely), or the GPU is incompatible.

**Comment by Kanopola** [comment]
> Did OCM work for you before the the internet connection was cut? This error suggest that the miner binaries are removed (by antivirus most likely), or the GPU is incompatible.

Yes, everything works normal, it only shows that when there is no internet. And what I ask is something as simple as mining again when it detects that the connection has returned.

In other words, when the Internet returns it begins to mine without the User's intervention; That is, the User does not have to click "retry"

Sí, todo funciona normal, solo muestra eso cuando no hay Internet. Y lo que pido es algo tan sencillo como que vuelva a minar cuando detecte que volvió la conexión.

O sea, que cuando vuelva el Internet empiece a minar sin intervención del Usuario; es decir, que el Usuario no tenga que darle clic a "reintentar"

**Comment by Kanopola** [comment]
> Effectively a duplicate of #408
> 
> Please keep issues in English.

¿Por qué de después de más de 2 años no lo han hecho?

Why haven't they done it after more than 2 years?

Estoy colocando las respuestas en ambos idiomas con el traductor de Google porque hay otros Usuarios de habla hispana en la comunidad; además, aunque sea una regla o protocolo, no es tan difícil usar un traductor.

I am putting the answers in both languages with Google translator because there are other Spanish-speaking Users in the community; Furthermore, even if it is a rule or protocol, it is not that difficult to use a translator.

**Comment by KforG** [comment]
> Why haven't they done it after more than 2 years?

Developers and maintainers are not paid, and thus has no obligation to do any work whatsoever. That means they work on what they want.

> I am putting the answers in both languages with Google translator because there are other Spanish-speaking Users in the community; Furthermore, even if it is a rule or protocol, it is not that difficult to use a translator.

I don't mind bilingual issues per say. About using a translator, the same goes they other way around.

This Github, and the issue tab is not for arguing about stuff like this.

**Comment by KforG** [comment]
> > Did OCM work for you before the the internet connection was cut? This error suggest that the miner binaries are removed (by antivirus most likely), or the GPU is incompatible.
> 
> Yes, everything works normal, it only shows that when there is no internet. And what I ask is something as simple as mining again when it detects that the connection has returned.
> 
> In other words, when the Internet returns it begins to mine without the User's intervention; That is, the User does not have to click "retry"
> 
> Sí, todo funciona normal, solo muestra eso cuando no hay Internet. Y lo que pido es algo tan sencillo como que vuelva a minar cuando detecte que volvió la conexión.
> 
> O sea, que cuando vuelva el Internet empiece a minar sin intervención del Usuario; es decir, que el Usuario no tenga que darle clic a "reintentar"

OCM doesn't always throw this error when internet connection is lost. In fact this is the first time I've seen it.
Normally (if mining has already been started), it'll automatically reconnect. 

Thanks for your bug report. I will try investigating this sometime. 
Could you turn on debugging in settings and provide the debug.log (found in `%appdata%\vertcoin-ocm on windows`), the next time you experience this?
